src/block.rs

```diff
@@ -552,61 +552,93 @@ impl Block {
 }
 
 #[derive(Copy, Clone, Debug, PartialEq)]
-pub enum Comparator {
+pub enum Operator {
```

Comment:
I don't feel this name change is warranted. All of them except None are comparators. Eq and QEq also do other things, but `Comparator` is a good way to describe them (and therefore vague)